extensions/measurement-tracking/src/panels/PanelStudyBrowserTracking/getStudiesForPatientByStudyInstanceUID.js

```diff
@@ -20,6 +20,9 @@ async function getStudiesForPatientByStudyInstanceUID(
     return dataSource.query.studies.search({
       patientId: getStudyResult[0].mrn,
     });
+  } else {
```

Comment:
mrn is sometimes null for anonymized data - I agree it is type 1, but some sample data has it that way, so support it.

Reply:
We don't need an `else` here

```js
if (getStudyResult && getStudyResult.length && getStudyResult[0].mrn) {
    return dataSource.query.studies.search({
      patientId: getStudyResult[0].mrn,
   });
}
 
console.log('No mrn found for', getStudyResult);
return getStudyResult;

}
```

Reply:
Also it shouldn't console.log `getStudyResult` as it contains whole study, maybe just UID?

Reply:
Also we have another one here `extensions/default/src/Panels/getStudiesForPatientByStudyInstanceUID.js`

Reply:
I know there is a second one - I didn't tweak that one as the behaviour is different enough I wasn't quite sure how to.